backend/templates/users/add_friend.html

```diff
@@ -0,0 +1,13 @@
+{% extends "base.html" %}
+{% block title %}Add Friend{% endblock %}
+
+{% block content %}
+<div class="container py-5">
+    <h1>Add a Friend</h1>
+    <form method="POST">
+        {% csrf_token %}
+        <input type="text" name="username" placeholder="Enter username or email" required>
+        <button type="submit" class="btn btn-primary">Send Friend Request</button>
```

Comment:
When a friend request is sent from here to the other user, the other user has no way of knowing they've received a request until they basically click on the "FRIENDS LIST" menu button as in they have to send a request to the backend

Reply:
we can add a badge on the Friend list when there is a pending requests.

Reply:
Done, please check.